internal/service/batch/job_queue_test.go

```diff
@@ -416,6 +454,53 @@ resource "aws_batch_job_queue" "test" {
 `, rName, priority))
 }
 
+func testAccBatchJobQueueSchedulingPolicy(rName string, rName2 string) string {
+	return fmt.Sprintf(`
+resource "aws_batch_scheduling_policy" "test1" {
+  name = %[1]q
+
+  fair_share_policy {
+    compute_reservation = 1
+    share_decay_seconds = 3600
+
+    share_distribution {
+      share_identifier = "A1*"
+      weight_factor    = 0.1
+    }
+  }
+}
+
+resource "aws_batch_scheduling_policy" "test2" {
+  name = %[2]q
+
+  fair_share_policy {
+    compute_reservation = 1
+    share_decay_seconds = 3600
+
+    share_distribution {
+      share_identifier = "A2"
+      weight_factor    = 0.2
+    }
+  }
+}
+`, rName, rName2)
+}
+
+func testAccBatchJobQueueConfigSchedulingPolicy(rName string, schedulingPolicyName1 string, schedulingPolicyName2 string, selectSchedulingPolicy string) string {
+	return acctest.ConfigCompose(
+		testAccBatchJobQueueConfigBase(rName),
+		testAccBatchJobQueueSchedulingPolicy(schedulingPolicyName1, schedulingPolicyName2),
+		fmt.Sprintf(`
+resource "aws_batch_job_queue" "test" {
+  compute_environments  = [aws_batch_compute_environment.test.arn]
+  name                  = %[1]q
+  priority              = 1
+  scheduling_policy_arn = "%[2]q" == "first" ? aws_batch_scheduling_policy.test1.arn : aws_batch_scheduling_policy.test2.arn
```

Comment:
While adding the quotes like `"%[2]q"` passes the `terrafmt` checks, it results in a failed test case with the following error:
```
=== CONT  TestAccBatchJobQueue_schedulingPolicy
    job_queue_test.go:144: Step 1/3 error: Error running pre-apply refresh: exit status 1
        
        Error: Missing newline after argument
        
          on terraform_plugin_test.tf line 128, in resource "aws_batch_job_queue" "test":
         128:   scheduling_policy_arn = ""first"" == "first" ? aws_batch_scheduling_policy.test1.arn : aws_batch_scheduling_policy.test2.arn
        
        An argument definition must end with a newline.
    testing_new.go:63: Error retrieving state, there may be dangling resources: exit status 1
        
        Error: Missing newline after argument
        
          on terraform_plugin_test.tf line 128, in resource "aws_batch_job_queue" "test":
         128:   scheduling_policy_arn = ""first"" == "first" ? aws_batch_scheduling_policy.test1.arn : aws_batch_scheduling_policy.test2.arn
        
        An argument definition must end with a newline.
        
--- FAIL: TestAccBatchJobQueue_schedulingPolicy (5.22s)
```

In this situation is there a way to ignore the `terrafmt` errors since without the quotes it allows for a successful test case but with the quotes it fails? Thanks!

Reply:
Resolved in a8e3982837bd0ea147aa9e7e7728b1f6c493b6e0 by referencing the value from `locals` instead of using it directly.

Test:

```
$ make testacc TESTS=TestAccBatchJobQueue_schedulingPolicy PKG=batch
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/batch/... -v -count 1 -parallel 20 -run='TestAccBatchJobQueue_schedulingPolicy' -timeout 180m
=== RUN   TestAccBatchJobQueue_schedulingPolicy
=== PAUSE TestAccBatchJobQueue_schedulingPolicy
=== CONT  TestAccBatchJobQueue_schedulingPolicy
--- PASS: TestAccBatchJobQueue_schedulingPolicy (223.66s)
PASS
ok      github.com/hashicorp/terraform-provider-aws/internal/service/batch      231.358s
```